test/pjrt/test_runtime_tpu.py

```diff
@@ -188,22 +188,22 @@ def test_spawn_threads(self):
           {i: torch.device(f'xla:{i}') for i in range(self.num_devices)})
 
   @staticmethod
-  def _device_attributes():
-    return xr.device_attributes(str(xm.xla_device()))
+  def _physical_device_attributes():
```

Comment:
Should these be `runtime` instead of `physical` in this file?

Reply:
good catch

Reply:
fixed.